specs/020-vm-cost-estimation/contracts/estimate_cost.proto

```
@@ -0,0 +1,51 @@
+// Contract reference for EstimateCost RPC.
+// This file documents the wire format; the actual proto is owned by finfocus-spec.
+// Do NOT compile this file — it is for documentation only.
+
+syntax = "proto3";
+
+package finfocus.v1;
```

Buf (1.65.0)

[error] Files with package "finfocus.v1" must be within a directory "finfocus/v1" relative to root but were in directory "specs/020-vm-cost-estimation/contracts".

(PACKAGE_DIRECTORY_MATCH)